### DISCUSSION TITLE
Home Assistant OS 10.1

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#2512] [Announcements]
## Home Assistant Operating System

* Bump buildroot to fix bluetoothctl segfault (#2509) @agners
* Enable Gasket driver for Google Coral support on ODROID-M1 (#2504) (#2508) @agners
* Avoid CLI restart on shutdown (#2507) @agners
* Bump buildroot to update systemd to 252.9 (#2506) @agners
* Fix qemu guest agent startup by using BindTo and After (#2496) (#2503) @agners

## Raspberry Pi

* Fix boot on Raspberry Pi 4 32-bit (#2500) (#2505) @agners
* Fix NVMe SSD boot for Raspberry Pi 4/CM4 (#2479) (#2493) @agners

## Open Virtual Appliance

* Fix startup by using BindTo and After (#2496) (#2503) @agners
* Linux: Update kernel 6.1.25 (#2495) @agners

## Generic x86-64

* Linux: Update kernel 6.1.25 (#2495) @agners

## Hardkernel ODROID

* Enable Gasket driver for Google Coral support on ODROID-M1 (#2504) (#2508) @agners
* Linux: Update kernel 6.1.25 (#2495) @agners

## ASUS Tinker

* Linux: Update kernel 6.1.25 (#2495) @agners

## Khadas VIM Series

* Enable HDMI on Khadas VIM3 (#2491) @agners

## Generic aarch64

* Linux: Update kernel 6.1.25 (#2495) @agners

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/10.1'>Home Assistant OS 10.1</a>.</em>

**Comment by ikke-zelf** [comment]
Any news on hardkernel building support for booting from NVMe with ODROID M1?
Or is that never going to happen?

**Reply by agners** [reply]
It seems unlikely at this point, as the U-Boot version Hardkernel's Petitboot is using is lacking proper PCIe/NVMe support in SPL.

**Reply by bcutter** [reply]
Again a release is shipped (stable environment) and it still contains the "beta release" remark. Remember seen this in the 9.x series some time ago - didn't we fix this (edit release description after releasing it from beta to stable)?

![image](https://user-images.githubusercontent.com/13799156/234921854-dfca70ef-7ba0-40b4-94b2-e9f5ea779ddf.jpeg)

**Reply by agners** [reply]
@bcutter release notes is still a manual thing, unfortunately :cry: Waiting for you PR :wink: 

Thanks for reminding me!

**Reply by ThWedum** [reply]
HA would not start after updating. "Wait Until Kernel Time Synchronized" fails on boot. Reverting to some older image that i had from before (8.5) resolved my issue. 
Running HA on a laptop

**Reply by agners** [reply]
That sounds like an internet connectivity issue: HAOS tries to synchronize time with an NTP server, by default with time.cloudflare.com at startup.

**Reply by ThWedum** [reply]
Any idea how the internet connectivity changes when i use different releases? Tried first updating to 10.1 trough HA. That is when it started. 
Also tried to clean install 10.1 to see if that did anything (but no difference). I can't wrap my head around this behavior.

**Reply by agners** [reply]
Temporary outage? Maybe systemd-resolved got more picky with DNS, and your local DNS/your providers DNS mangles requests such that systemd-resolved doesn't like it?

Honestly, not sure what this is, but we have 70k plus users where it seems to work... So it must some local environment + HAOS 10.1 interaction issue...

**Reply by ThWedum** [reply]
Tried this over the span of three days before giving up and downgrading. I understand that this might be a edge case. I don’t expect anyone to dedicate time to my issue as long as I’m the only one reporting it. Still thought it could be important to write it down just in case that I’m not alone.

**Comment by mcpingvin** [comment]
Is it just me, or did this update break Tuya integration?

After re-adding the integration since it is just gone (with all cloud and user data), when adding a Tuya device it asks for the credentials again and doesn't offer any devices.

**Reply by agners** [reply]
This is very unlikely HAOS related, as it just provides the basis to run Home Assistant Core. Maybe you did a HA Core update at the same time?

If you still suspect HAOS, you can downgrade it back to `ha os update --version 9.5` to verify if that resolves the issue.

**Reply by mcpingvin** [reply]
No, I've just updated it and now and it's when the problem started. Core was updated a few days ago.

I'll try to downgrade and comment back.

**Reply by mcpingvin** [reply]
I think I've found the culprit, expired trial: https://github.com/home-assistant/core/issues/91654#issuecomment-1515860585

Thanks for the help!
